catalog/spatial/geocoding/spatial-geocoding-geocoder/src/main/java/org/codice/ddf/spatial/geocoder/GazetteerGeoCoder.java

```diff
@@ -75,19 +73,10 @@ public NearbyLocation getNearbyCity(String location) throws GeoEntryQueryExcepti
   @Override
   public Optional<String> getCountryCode(String locationWKT, int radius) {
     try {
-      Optional<String> alpha2CountryCode = geoEntryQueryable.getCountryCode(locationWKT, radius);
-
-      if (alpha2CountryCode.isPresent()) {
-        try {
-          String alpha3CountryCode =
-              new Locale(Locale.ENGLISH.getLanguage(), alpha2CountryCode.get()).getISO3Country();
-          return Optional.of(alpha3CountryCode);
-        } catch (MissingResourceException e) {
-          LOGGER.debug(
-              "Failed to convert country code {} to alpha-3 format. Returning empty value",
-              alpha2CountryCode.get(),
-              e);
-        }
+      Optional<String> alpha3CountryCode = geoEntryQueryable.getCountryCode(locationWKT, radius);
```

Comment:
you could simplify to `return geoEntryQueryable.getCountryCode(locationWKT, radius);`